packages/kbn-content-management/table_list/src/table_list_view/mocks.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { Services } from './services';
+
+/**
+ * Parameters drawn from the Storybook arguments collection that customize a component story.
+ */
+export type Params = Record<keyof ReturnType<typeof getStoryArgTypes>, any>;
+type ActionFn = (name: string) => any;
+
+/**
+ * Returns Storybook-compatible service abstractions for the `NoDataCard` Provider.
+ */
+export const getStoryServices = (params: Params, action: ActionFn = () => {}) => {
```

Comment:
It's up to you, but you might want to use the new Storybook mocks approach:

https://github.com/elastic/kibana/blob/main/packages/shared-ux/card/no_data/mocks/src/storybook.ts

You can leave it as a todo for a future PR, though.

Reply:
Ok. Probably for a future PR 👍